index.html

```diff
@@ -76,132 +82,117 @@ <h3 class="H-title">I'am a Software Developer</h3>
                 </div>
               </section>
 
-              <!--Projects section-->
-              <section id="projects">
-                <div id="h2-container">
-                  <h2 class="h2-project">My Recent Works</h2>
-                  <div id="h2-div"></div>
-                </div>
-                <div id="portfolio-grid">
-                  <div id="main-project">
-                    <div id="main-image-div">
-                      <img
-                        src="./images/Img.png"
-                        alt="project-works"
-                        id="project-works"
-                        data-img="image"
-                      />
-                    </div>
-                    <div id="project-content">
-                      <h3 id="project-h3" data-title="title">Multi-Post Stories</h3>
-                      <p id="content-p">
-                        A daily selection of privately personalized reads; no accounts or
-                        sign-ups required. has been the industry's standard dummy text
-                        ever since the 1500s, when an unknown printer took a standard
-                        dummy text. A daily selection of privately personalized reads; no
-                        accounts or sign-ups required.
-                      </p>
-                      <div>
-                      <ul class="language-button12">
-                        <li>html</li>
-                        <li>css</li>
-                        <li>bootstrap</li>
-                        <li>Ruby</li>
-                      </ul>
-                      <button
-                        type="button"
-                        id="main-project-button"
-                        class="btn project-btn"
-                        data-btn="#main-project">
-                        See Project
-                      </button>
-                    </div>
-                    </div>
-                  </div>
-                </div>
-              </section>
+<section class="proje1">
+  <div class="roject__recent">
+    <h2 class="recent__h2">My Recent Works<span class="recent__h2__line"></span></h2>
+      </div>
+    <div class="work">
 
-              <!--Some work section-->
-              <section id="container-grid">
-                <div class="card-work">
-                  <h1 class="papd">Professional Art<br>Printing Data</h1>
-                  <p class="papd-p">Profesional Art Printing Data
-                    A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-                   <ul class="language-button1">
-                      <li>html</li>
-                      <li>bootstrap</li>
-                      <li>Ruby</li>
-                    </ul>
-                    <button 
```

Comment:
- [ ] Kindly, Consider adding a background color to your languages as you can see to the images down ⬇️ 

- Figma design:

![good2](https://user-images.githubusercontent.com/79058364/161404745-dc4869b3-d1ac-4aa6-a8f2-e50283ba8f68.png)


- Your design:

![bad2](https://user-images.githubusercontent.com/79058364/161404751-d5997bad-a6aa-4b84-bd16-5088f3801a6b.png)